docs/README.md

```diff
@@ -0,0 +1,54 @@
+# LlamaIndex Documentation
+
+## A guide for docs contributors
+
+The `docs` directory contains the sphinx source text for LlamaIndex docs, visit
+https://gpt-index.readthedocs.io/ to read the full documentation.
+
+This guide is made for anyone who's interested in running LlamaIndex documentation locally,
+making changes to it and make contributions. LlamaIndex is made by the thriving community
+behind it, and you're always welcome to make contributions to the project and the 
+documentation. 
+
+## Build Docs
+
+If you haven't already, clone the LlamaIndex Github repo to a local directory:
+
+```bash
+git clone https://github.com/jerryjliu/llama_index.git && cd llama_index
+```
+
+Install all dependencies required for building docs (mainly `sphinx` and its extension):
+
+```bash
+pip install -r docs/requirements.txt
+```
+
+Build the sphinx docs:
+
+```bash
+cd docs
+make html
+```
+
+The docs HTML files are now generated under `docs/_build/html` directory, you can preview
+it locally with the following command:
```

Comment:
another simple way to run is just open it as an html web page e.g. `open _build/html/index.html`